pkg/workers/updates/updates.go

```diff
@@ -1,110 +1,70 @@
-package instance
+package updates
 
 import (
-	"context"
 	"fmt"
 	"net/url"
 	"strings"
 	"sync"
 	"time"
 
 	"github.com/cozy/cozy-stack/pkg/apps"
-	"github.com/cozy/cozy-stack/pkg/config"
-	"github.com/cozy/cozy-stack/pkg/logger"
+	"github.com/cozy/cozy-stack/pkg/instance"
+	"github.com/cozy/cozy-stack/pkg/jobs"
 	"github.com/cozy/cozy-stack/pkg/registry"
-	"github.com/cozy/cozy-stack/pkg/utils"
 	multierror "github.com/hashicorp/go-multierror"
-	"github.com/robfig/cron"
 )
 
-const numUpdaters = 50
+const numUpdaters = 25
```

Comment:
Not for this PR, but shouldn't the number of updaters be configurable?

Reply:
Yes, and I'll remove this diff.